yb-voyager/src/dbzm/dbzm.go

```diff
@@ -0,0 +1,92 @@
+package dbzm
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+var DEBEZIUM_DIST_DIR, DEBEZIUM_CONF_DIR, DEBEZIUM_CONF_FILEPATH string
+
+type Debezium struct {
+	*Config
+	cmd  *exec.Cmd
+	err  error
+	done bool
+}
+
+func NewDebezium(config *Config) *Debezium {
+	return &Debezium{Config: config}
+}
+
+func initVars() {
+	log.Infof("initializing debezium env variables like DEBEZIUM_DIST_DIR...")
+	// take value of DEBEZIUM_DIST_DIR value from environment variables
+	if envVal := os.Getenv("DEBEZIUM_DIST_DIR"); envVal != "" {
+		DEBEZIUM_DIST_DIR = envVal
+	} else {
+		utils.ErrExit("DEBEZIUM_DIST_DIR environment variable is not set")
+	}
+
+	DEBEZIUM_CONF_DIR = filepath.Join(DEBEZIUM_DIST_DIR, "conf")
+	DEBEZIUM_CONF_FILEPATH = filepath.Join(DEBEZIUM_CONF_DIR, "application.properties")
+}
+
+func (d *Debezium) Start() error {
+	initVars()
```

Comment:
You can use `init() {}` function to initialise these variables. That way you don't need to explicitly call the init.